backend-rust/src/migrations/m0020_amounts_schedule.rs

```diff
@@ -0,0 +1,83 @@
+use super::{SchemaVersion, Transaction};
+use crate::transaction_event::events_from_summary;
+use anyhow::Context;
+use async_graphql::futures_util::StreamExt;
+use concordium_rust_sdk::{
+    types::{
+        AbsoluteBlockHeight, AccountTransactionEffects, BlockItemSummaryDetails, TransactionType,
+    },
+    v2::{self},
+};
+
+/// Performs a migration that creates and populates the baker metrics table.
+pub async fn run(
+    tx: &mut Transaction,
+    endpoints: &[v2::Endpoint],
+    next_schema_version: SchemaVersion,
+) -> anyhow::Result<SchemaVersion> {
+    let endpoint = endpoints.first().context(format!(
+        "Migration '{}' must be provided access to a Concordium node",
+        next_schema_version
+    ))?;
+
+    let mut client = v2::Client::new(endpoint.clone()).await?;
+
+    let rows = sqlx::query(
+        "
+            SELECT
+                block_height,
+                COUNT(*) AS update_count
```

Comment:
This will also be used.

Reply:
I don't see that this is used.

Reply:
It is not, I will make the change was what I meant. I was busy with fixing something else.

Reply:
It was not as trivially calculated as before (Type and effect can differ), and will remove it to remove noise.